stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/dao/ChannelConfigDao.kt

```diff
@@ -4,19 +4,37 @@ import androidx.room.Dao
 import androidx.room.Insert
 import androidx.room.OnConflictStrategy
 import androidx.room.Query
+import androidx.room.Transaction
 import io.getstream.chat.android.livedata.entity.ChannelConfigEntity
+import io.getstream.chat.android.livedata.entity.ChannelConfigInnerEntity
+import io.getstream.chat.android.livedata.entity.CommandInnerEntity
 
 @Dao
 internal interface ChannelConfigDao {
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
-    suspend fun insertMany(channelConfigEntities: List<ChannelConfigEntity>)
+    @Transaction
+    suspend fun insert(channelConfigEntities: List<ChannelConfigEntity>) {
+        channelConfigEntities.map(ChannelConfigEntity::channelConfigInnerEntity).also { insertConfigs(it) }
+        channelConfigEntities.flatMap(ChannelConfigEntity::commands).also { insertCommands(it) }
```

Comment:
Nit: I'd use `let` here to make use of the value, `also` is usually used when its return value is used in some way ("give me this object, but _also_ do stuff before that")

Reply:
Look at the latest commit. IDE suggested another :-)

Reply:
Well, yes, that's the boring way I'd have gone with to begin with, but I wanted to give a suggestion given that you chose the functional style already 😄